mirgecom/boundary.py

```diff
@@ -57,32 +147,33 @@ def __init__(self, userfunc):
             least one parameter that specifies the coordinates at which to prescribe
             the solution.
         """
-        self._userfunc = userfunc
-
-    def boundary_pair(self, discr, cv, btag, **kwargs):
-        """Get the interior and exterior solution on the boundary."""
-        actx = cv.array_context
-
-        boundary_discr = discr.discr_from_dd(btag)
-        nodes = thaw(actx, boundary_discr.nodes())
-        ext_soln = self._userfunc(nodes, **kwargs)
-        int_soln = discr.project("vol", btag, cv)
-        return TracePair(btag, interior=int_soln, exterior=ext_soln)
+        from warnings import warn
+        warn("Do not use PrescribedBoundary; use PrescribedInvscidBoundary. This"
+             "boundary type  will disappear soon.", DeprecationWarning, stacklevel=2)
```

Comment:
Put a deadline on it.

Reply:
The reason I made it so tight is we will likely re-use this boundary type in our boundary refactor effort.  We've already gone around and updated all the examples and production stuff.  Nobody out there should be still using this boundary.